tests/integration_tests/test_ray_tracing.py

```diff
@@ -165,34 +153,3 @@ def test_integral_curve(db_config, simtel_path_no_mock, io_handler, telescope_mo
         file_name="test_cumulative_psf.pdf", sub_dir="plots", dir_type="test"
     )
     plt.savefig(plot_file)
-
-
-def test_process_rx(
```

Comment:
I think you removed this because you replaced it with a mock unit test. However, this would actually call rx (and fail), which is a different test (and an integration one). Would the mocking test in the unit tests actually call rx? Should we rethink this removal?

Reply:
What I see is that `test_process_rx_valid_output` mocks the call to `./sim_telarray/bin/rx` (I think). If so, we are not really calling `rx`. That is good in the unit tests, we do not want to call sim_telarray there. But then where is the integration test that actually calls `rx`?

Reply:
is it possible that our confusion comes from that your are looking int `tests/unit_tests/test_ray_tracing.py`, while I am looking into `tests/integration_tests/test_ray_tracing.py`?

This is very I think the integration tests calls `rx`: https://github.com/gammasim/simtools/blob/5aaef263b962bb0a07cf8490746f47aaf36543ba/tests/integration_tests/test_ray_tracing.py#L39

Reply:
Yes, that's the source of the confusion, apologies.